subiquity/models/subiquity.py

```diff
@@ -93,6 +93,14 @@ def setup_yaml():
 ff02::2 ip6-allrouters
 """
 
+CLOUDINIT_CLEAN_FILE_TMPL = """\
+#!/bin/sh
+# Remove live-installer config artifacts when running: sudo cloud-init clean
+for cfg_file in {cfg_files};
+  rm -f $cfg_file;
```

Comment:
Are you concerned with the return code of this script?
I ask because if there are multiple files, you will only see the return code of the last `rm`.  Maybe that's OK.

Reply:
Wasn't concerned with the failure of multiple files on rm -f as the force flag should get us past any files which are absent from previous runs of this script.  I've shifted the implementation to python per part of @ogayot's suggestion.